tests/conftest.py

```
@@ -0,0 +1,147 @@
+"""Shared pytest fixtures for helm-charts-updater tests."""
+
+import os
+from pathlib import Path
+from typing import Any
+from typing import Dict
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)